mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/validator/marc/impl/MarcFieldNonRepeatableRequired1xxFieldRuleValidator.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.rspec.validation.validator.marc.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.folio.rspec.domain.dto.SpecificationDto;
+import org.folio.rspec.domain.dto.ValidationError;
+import org.folio.rspec.i18n.TranslationProvider;
+import org.folio.rspec.validation.validator.SpecificationRuleCode;
+import org.folio.rspec.validation.validator.marc.model.MarcField;
+import org.folio.rspec.validation.validator.marc.model.MarcRuleCode;
+import org.folio.rspec.validation.validator.marc.utils.TagsMatcher;
+
+class MarcFieldNonRepeatableRequired1xxFieldRuleValidator
+  extends Abstract1xxFieldRuleValidator {
+
+  MarcFieldNonRepeatableRequired1xxFieldRuleValidator(TranslationProvider translationProvider) {
+    super(translationProvider);
+  }
+
+  @Override
+  public List<ValidationError> validate(Map<String, List<MarcField>> fields, SpecificationDto specification) {
+    var errors = new ArrayList<ValidationError>();
+    if (fields.keySet().stream().noneMatch(TagsMatcher::matches1xx)) {
```

Comment:
Shouldn't this validation return an error also if there are multiple 1XX?

Reply:
Yep, will add it after MRSPECS-41 fixing

Reply:
Fixed